symphony-bdk-core/src/main/java/com/symphony/bdk/core/config/model/BdkClientConfig.java

```diff
@@ -38,26 +35,45 @@ public boolean overridesParentConfig() {
     return scheme != null || host != null || port != null || context!= null;
   }
 
+  @Override
   public String getScheme() {
     return thisOrParent(scheme, parentConfig::getScheme);
   }
 
+  @Override
   public String getHost() {
     return thisOrParent(host, parentConfig::getHost);
   }
 
+  @Override
   public Integer getPort() {
     return thisOrParent(port, parentConfig::getPort);
   }
 
+  @Override
   public String getContext() {
     return thisOrParent(context, parentConfig::getContext);
   }
 
-  public BdkProxyConfig getProxy() {
-    return thisOrParent(proxy, parentConfig::getProxy);
+  @Override
+  public Integer getConnectionTimeout() {
+    return thisOrParent(connectionTimeout, parentConfig::getConnectionTimeout);
+  }
+
+  @Override
+  public Integer getReadTimeout() {
+    return thisOrParent(readTimeout, parentConfig::getReadTimeout);
   }
 
+  @Override
+  public Integer getConnectionPoolMax() { return thisOrParent(connectionPoolMax, parentConfig::getConnectionPoolMax); }
+
+  @Override
+  public Integer getConnectionPoolPerRoute() { return thisOrParent(connectionPoolPerRoute, parentConfig::getConnectionPoolPerRoute); }
+
+  @Override
+  public BdkProxyConfig getProxy() { return thisOrParent(proxy, parentConfig::getProxy); }
```

Comment:
Our [`.editorconfig`](https://github.com/SymphonyPlatformSolutions/symphony-api-client-java/blob/master/.editorconfig#L133) should prevent for simple methods in 1 line

Reply:
changed, but they was a copy-paste of methods already formatted in this way. I will push it later